src/posh/reagent.cljc

```diff
@@ -5,6 +5,11 @@
             [reagent.core :as r]
             [reagent.ratom :as ra]))
 
+(defn derive-reaction [reactions key f & local-mixin]
+  (apply ra/make-reaction
```

Comment:
Hello @bamarco . I tried to manually recreate your PR, but run into the issue. The thing is, it looks like there is no such thing as `reagent.ratom/make-reaction` in the clojure version of namespace https://github.com/reagent-project/reagent/blob/master/src/reagent/ratom.clj
So when I tried to run tests with this code they failed to compile with a message:
> Syntax error compiling at (posh/reagent.cljc:9:3).
> No such var: ra/make-reaction

Could you please have a look at this?